ui/src/components/styles.ts

```diff
@@ -61,6 +61,10 @@ export const TableWrapper = styled.div`
   margin-top: ${gridSize() * 4}px;
   max-height: 60vh;
   overflow: auto;
+  &::-webkit-scrollbar {
```

Comment:
Why did the scrollbar need to be restyled?

Reply:
Updated table styles and removed this code.